tests/VCR/VCRCleanerEventSubscriberTest.php

```diff
@@ -183,4 +183,51 @@ public function testCurlCallWithRedactedHostname()
         $this->assertNotContains('www.example.com', $vcrFile);
         $this->assertContains('[redacted]', $vcrFile);
     }
+
+    public function testCurlCallToModifyResponseHeaders()
+    {
+        $newFile = $this->getCassetteContent();
+        $this->assertEmpty($newFile);
+
+        VCRCleaner::enable(array(
+            'ignoreResponseHeaders' => array(
+                'X-Powered-By',
+            ),
+        ));
+
+        $curl = new Curl();
+        $curl->get('https://reqres.in/api/users/2');
```

Comment:
Can I avoid calling an external API? Can php-vcr be hooked up into here **without** breaking the actual unit test functionality...?

Reply:
I'll just stick with `example.com` which doesn't throw any errors.